my.css

```diff
@@ -0,0 +1,270 @@
+body {
+	margin: 0;
+	padding: 0;
+}
+
+.header .header {
+	background: black;
+	position: relative;
+}
+
+.logo {
+	position: absolute;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5556249)